### PR TITLE
Db2 license

### PR DESCRIPTION
**Pull request by tnixon**
Moving to new DB license so that we can accept external contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,5 +18,18 @@ a suite of abstractions and functions that make operations on timeseries data ea
...
+## Contributing
+We happily welcome contributions to Tempo. 
+We use [GitHub Issues](https://github.com/databrickslabs/tempo/issues) to track community reported issues 
+and [GitHub Pull Requests](https://github.com/databrickslabs/tempo/pulls) for reviewing and accepting code changes.
+
+If you have found a bug or other issue with the library, please [open an new issue](https://github.com/databrickslabs/tempo/issues/new) to report it.
+If you have written a patch for a bug or new feature, please [submit a pull request](https://github.com/databrickslabs/tempo/compare) and we will be happy to review it for inclusion in a future release.
+
+You can [create a fork of the code](https://github.com/databrickslabs/tempo/fork) to begin your own modifications. 
+More comprehensive instructions on setting up and building the project can be found in [BUILDING.md](BUILDING.md) and [CONTRIBUTING.md](CONTRIBUTING.md).
```
Comment (R7L208) [inline README.md:33]: Do we need this here and in `CONTRIBUTING.md`?

**Review by R7L208 — COMMENTED** [review]
If possible, let's add new contribution details in README.md or CONTRIBUTING.md but not both. Maybe in the README we can just state that we welcome contributions and details are in the CONTRIBUTING file?

**Comment by tnixon** [comment]
Yes, that makes sense. Do you think we can consolidate Build.md into CONTRIBUTING.md? Just have a single place for all setup and build instructions.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+We happily welcome contributions to Tempo. 
+We use [GitHub Issues](https://github.com/databrickslabs/tempo/issues) to track community reported issues 
+and [GitHub Pull Requests](https://github.com/databrickslabs/tempo/pulls) for reviewing and accepting code changes.
+
+If you have found a bug or other issue with the library, please [open an new issue](https://github.com/databrickslabs/tempo/issues/new) to report it.
+If you have written a patch for a bug or new feature, please [submit a pull request](https://github.com/databrickslabs/tempo/compare) and we will be happy to review it for inclusion in a future release.
+
+You can [create a fork of the code](https://github.com/databrickslabs/tempo/fork) to begin your own modifications. 
+
+## Tox Setup instructions
```
Comment (rportilla-databricks) [inline CONTRIBUTING.md:11]: Shouldn't tox be in the README.md and documentation? I think pandas is a good example of placing build instructions in the official docs with BSD 3 license strictly having terms for contribution. Separately, building is up to the tempo project and should be in the github workflows (e.g. https://github.com/pandas-dev/pandas/blob/main/.github/workflows/wheels.yml)

**Review by rportilla-databricks — CHANGES_REQUESTED** [review]

**Pull request closed without merging**